gdnative-core/src/core_types/vector2.rs

```diff
@@ -34,6 +35,8 @@ pub trait Vector2Godot {
     fn from_sys(v: sys::godot_vector2) -> Self;
 }
 
+const CMP_EPSILON: f32 = 0.00001;
```

Comment:
Maybe use `approx` (currently an optional dependency for tests) instead of inventing your own constant?

Reply:
changed